packages/discord.js/typings/index.d.ts

```diff
@@ -9,7 +9,7 @@ import {
   StringSelectMenuBuilder as BuilderStringSelectMenuComponent,
   UserSelectMenuBuilder as BuilderUserSelectMenuComponent,
   TextInputBuilder as BuilderTextInputComponent,
-  SelectMenuOptionBuilder as BuildersSelectMenuOption,
+  StringSelectMenuOptionBuilder as BuildersSelectMenuOption,
```

Comment:
This isn't accurate anymore right? ALSO we should probs not edit _anything_ in djs as it'll get picked up as major